supplements/sword-coast-adventurers-guide/backgrounds/background-mercenaryveteran.xml

```diff
@@ -24,8 +24,8 @@
       <p>Your bond could be associated with the company you traveled with previously, or with some of the comrades you served with. The ideal you embrace largely depends on your worldview and your motivation for fighting.</p>
       <h5>MERCENARIES OF THE NORTH</h5>
       <p>Countless mercenary companies operate up and down the Sword Coast and throu ghout the North. Most are smallsca le opera tions that employ a dozen to a hundred folk who offer secu rity services, hunt monsters and brigands, or go to war in exchange for gold. Some organizations, such as the Zhentarim, Flaming Fist, and the nation of Mintarn have hundreds or thousands of members and can provide private armies to those with enough funds. A few organizations operating in the North are described below.</p>
-      <p class="indent"><span class="feature">The Chill.</span>The cold and mysterious Lurkwood serves as the home of numerous groups of goblinoids that have banded together into one tribe ca ll ed the Chill. Unlike most of their kind, the Chill refrains from raiding the people of the North and maintains relatively good relations so that they can hire themselves out as warriors. Few ci tystates in the North are willing to field an army alongside the Chill, but several are happy to qu ietly pay the Chill to battle the Uthgardt, ores, trolls of the Evermoors, and other threats to civilization.</p>
-      <p class="indent"><span class="feature">Silent Rain.</span>Consisting solely of elves, Silent Rain is a legendary mercenary company operating out of Evereska. Caring little for gold or fame, Silent Rain agrees only to jobs that either promote elven causes or invo lve destroying ores, gnolls, and the like. Prospective employers must leave written word (in Elvish) near Evereska, and the Silent Rain sends a representative if interested.</p>
+      <p class="indent"><span class="feature">The Chill.</span>The cold and mysterious Lurkwood serves as the home of numerous groups of goblinoids that have banded together into one tribe ca ll ed the Chill. Unlike most of their kind, the Chill refrains from raiding the people of the North and maintains relatively good relations so that they can hire themselves out as warriors. Few ci tystates in the North are willing to field an army alongside the Chill, but several are happy to qu ietly pay the Chill to battle the Uthgardt, orcs, trolls of the Evermoors, and other threats to civilization.</p>
```

Comment:
Could you please also fix this one: "goblinoids that have banded together into one tribe **ca ll ed** the Chill"

Reply:
Done.